solutions/silver/cc-TRPTSTIC.mdx

```diff
@@ -119,4 +119,96 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```py
+def rangesum(row1: int, column1: int, row2: int, column2: int) -> int:
+	"""
+	:return the sum from (row1, column1) to (row2, column2)
+	Assumes row1 < row2 and column1 < column2
```

Comment:
add an `assert` instead of just putting it in the docs

Reply:
done.